### ISSUE TITLE
Everything is back to London

### ISSUE THREAD

**Issue by jidanni** [umap-project/umap#1694]
https://umap.openstreetmap.fr/en/map/wilmette-il-usa-house-addressing-irregularities_961179
Should go to Wilmette IL, but instead just goes to London.

Workaround: never trust the share buttons, instead construct your own URL, https://umap.openstreetmap.fr/en/map/wilmette-il-usa-house-addressing-irregularities_961179?share#14/42.0779/-87.7087 that has a coordinate reference on the back.

The problem is just this one file, it turns out. But sorry, I can't figure out why it flies to London.

**Comment by PeskyPotato** [comment]
Hi @jidanni I took a look at the map you linked, after exporting the entire map properties I saw the following
```
    "geometry": {
        "type": "Point",
        "coordinates": [
            2,
            51
        ]
    }
 ```
 
 Which, from what I understand, are the coordinates used to center the map. Those lie on the straight of Dover between the UK and France, so lines up roughly with what you say.

Have you tried using the "Save this center and zoom" button outlined in red and then saved the map? That button changed the coordinates mentioned above to roughly `42.07, -87.73` which is in Lake Michigan.

![bitmap](https://github.com/umap-project/umap/assets/35942108/1108c25a-36cb-48cb-a939-14245b375a3b)

---

On a side note, both links you shared above opened at Evanston, IL for me and not London. So the issue might in fact be something else if the above doesn't work.
